Dockerfiles/Dockerfile.test

```diff
@@ -0,0 +1,34 @@
+FROM rust:1.46 as builder
+
+RUN rustup target install aarch64-unknown-linux-musl
+WORKDIR /tmp/workdir
+COPY ./nsm-lib nsm-lib
+COPY ./nsm-io nsm-io
+COPY ./nsm-driver nsm-driver
+COPY ./nsm-test nsm-test
+COPY ./Cargo.toml Cargo.toml
+## command-executer binary is required from aws-nitro-enclaves-cli repository
+COPY ./command-executer/command_executer_docker_dir/command-executer command-executer
+RUN cargo build
+RUN cargo install --path nsm-test
+RUN strip /usr/local/cargo/bin/nsm-check
+RUN cd nsm-test/ && make
+
+FROM public.ecr.aws/debian/debian:buster-slim as nsm-check
```

Comment:
If we build with musl, we might as well do FROM scratch here.

Reply:
I think command-executer uses /bin/sh or /bin/bash for launching commands, so this would require adding a few packages to the Docker image I think.